bot.py

```
@@ -0,0 +1,610 @@
+
+        fighterIDs = []
+        inFight = False
+
+    read()
+    fight = False
+    inFight = False
+    await client.get_channel(901363239605116989).send("We're live! Be prepared for random battles!")
+    await client.get_channel(901363239605116989).send(file=discord.File('./images/bonjour.gif'))
+
+    while(True):
+        await asyncio.sleep(random.random() * 1000 + 120)
```

Ruff (0.14.0)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)